playbooks/ansible-cloud/terraform/pre.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: all
+  tasks:
+    - name: Get current Terraform version
+      uri:
+        url: https://checkpoint-api.hashicorp.com/v1/check/terraform
```

Comment:
There is also a RPM repository: https://developer.hashicorp.com/terraform/cli/install/yum it may easier to follow this path.

Reply:
Ah, yeah, thanks.